### ISSUE TITLE
Summary Box Tile (Proxy)

### ISSUE THREAD

**Issue by robgietema** [plone/volto#288]

**Comment by sneridagh** [comment]
This will be superseded by the upcoming Teaser Block.

https://github.com/plone/volto/pull/3180

**Comment by sneridagh** [comment]
@plone/volto-team Finally, we can sleep after #3706 is merged 😎 🎉
